targets/switch-test/switch_test.cpp

```diff
@@ -0,0 +1,121 @@
+/**
+ * This code will allow for an easy testing of the APM MOSFET switches
+ */
+
+#include <EVT/io/manager.hpp>
+#include <EVT/io/pin.hpp>
+#include <EVT/io/UART.hpp>
+#include <APM/APMUart.hpp>
+#include <cstring>
+#include <APM/APMManager.hpp>
+#include <string>
+
+#pragma clang diagnostic push
+#pragma ide diagnostic ignored "EndlessLoop"
+
+namespace IO = EVT::core::IO;
+
+namespace APM {
+
+constexpr int BAUD_RATE = 115200;
+constexpr int BUF_SIZE = 256;
+
+#ifdef NUCLEO_COMPILATION
+constexpr IO::Pin UART_TX = IO::Pin::UART_TX;
+constexpr IO::Pin UART_RX = IO::Pin::UART_RX;
+#else
+constexpr IO::Pin UART_TX = IO::Pin::PB_10;
+constexpr IO::Pin UART_RX = IO::Pin::PB_11;
+#endif
+
+char buf[BUF_SIZE];
+
+}  // namespace APM
+
+/**
+ * Returns the inverse of the current GPIO state
+ * @param state the current state of a GPIO pin
+ * @return the opposite state
+ */
+EVT::core::IO::GPIO::State get_toggle(EVT::core::IO::GPIO::State state) {
```

Comment:
Should be camel case for style (Can bring in the clang-format support from the project template).